SU2_CFD/src/solvers/CScalarSolver.cpp

```diff
@@ -904,8 +904,13 @@ void CScalarSolver::LoadRestart(CGeometry **geometry, CSolver ***solver, CConfig
   solver[MESH_0][SCALAR_SOL]->InitiateComms(geometry[MESH_0], config, SOLUTION);
   solver[MESH_0][SCALAR_SOL]->CompleteComms(geometry[MESH_0], config, SOLUTION);
 
+  // Flow-Pre computes/sets mixture properties
   solver[MESH_0][FLOW_SOL]->Preprocessing(geometry[MESH_0], solver[MESH_0], config, MESH_0, NO_RK_ITER, RUNTIME_FLOW_SYS, false);
-  solver[MESH_0][SCALAR_SOL]->Postprocessing(geometry[MESH_0], solver[MESH_0], config, MESH_0);
+  // Update eddy-visc which needs correct mixture density and mixture lam-visc. Note that after this, another Flow-Pre
+  // at the start of the Iteration sets the updated eddy-visc into the Flow-Solvers Primitives.
+  if(config->GetKind_Turb_Model()) solver[MESH_0][TURB_SOL]->Postprocessing(geometry[MESH_0], solver[MESH_0], config, MESH_0);
+  // For feature_multicomp this Scalar-Pre only computes the laminar contribution to mass diffusivity
+  solver[MESH_0][SCALAR_SOL]->Preprocessing(geometry[MESH_0], solver[MESH_0], config, MESH_0, NO_RK_ITER, RUNTIME_FLOW_SYS, false);
```

Comment:
Note that Turb-Post (computes `eddy-visc(rho_mix, mu_lam_mix)`) and Scalar-Pre (computes `lam_Diffusivity(rho_mix, cp_mix, therm_cond_mix)`) can be switched.